api/test/test_play_asset_pack_manager.gd

```diff
@@ -161,3 +162,117 @@ func test_cancel_asset_pack_request_non_existing_pack_name():
 	
 	var test_result : bool = test_object.cancel_asset_pack_request(test_pack_name)
 	assert_eq(test_result, false)
+
+func test_show_cellular_data_confirmation_success():
+	var mock_plugin = FakeAndroidPlugin.new()
+
+	# configure what should be emitted upon show_cellular_data_confirmation() call
+	mock_plugin.set_show_confirmation_response(true, PlayAssetPackManager.AssetPackStorageMethod.STORAGE_FILES, {})
+	var test_object = create_play_asset_pack_manager(mock_plugin)
+	
+	var request_object = test_object.show_cellular_data_confirmation()
+	
+	# connect to the helper function, simulating the workflow connecting callback to signal
+	request_object.connect("request_completed", self, "show_cellular_data_confirmation_success_helper")
+	
+	# yield to the request_completed signal for no longer than 1 seconds and assert for signal emitted
+	yield(yield_to(request_object, "request_completed", 1), YIELD)
```

Comment:
This [syntax](https://github.com/bitwes/Gut/wiki/Yielding) for yield is specifically used for Gut testing environment.